docs/using.md

```diff
@@ -210,12 +219,37 @@ def callback_handler (stream_consumer: qx.StreamConsumer, data: qx.TimeseriesDat
 ...
 ```
 
+!!! warning
+    With the previous example code, you should note that the running total will not be preserved in the event of system crashes or restarts.
+
 The key point here is that data is tracked per stream context. You keep running totals on a per-stream basis by using the stream ID, `stream_consumer.stream_id` to index a dictionary containing running totals for each stream.
 
 !!! note
 
     A stream will only ever be processed by one replica.
 
+## Handling system restarts and crashes
+
+One issue you may run into is that in-memory data is not persisted across instance restarts, shutdowns, and instance crashes. This can be mitigated by using the Quix Streams `LocalFileStorage` facility. This will ensure that specified variables are persisted on permanent storage, and this data is preserved across restarts, shutdowns, and system crashes.
+
+The following example code demonstrates a simple use of `LocalFile Storage`:
+
+```python
```

Comment:
Can you here provide a full code sample as this is actually what they looking for? Also, can you make sure it is very visible this is the target code.

Reply:
I've extended the example.